src/map/script.c

```diff
@@ -2861,32 +2861,44 @@ struct script_data *get_val(struct script_state* st, struct script_data* data) {
 		const char *str = NULL;
 
 		switch (prefix) {
-			case '@':
+		case '@':
+			if (data->ref) {
+				str = script->get_val_ref_str(st, data->ref, data);
```

Comment:
Never ever re-indent a block AND change some lines in it in the same commit. It makes it really hard to review the changes and/or merge them for users.